apps/system/style/cards_view/cards_view.css

```diff
@@ -75,23 +85,37 @@
 /* Cards */
 
 #cards-view .card {
+  display: block;
   position: absolute;
-  top: 26%;
-  left: 26%;
-  width: 48%;
-  height: 48%;
+  top: 0;
+  left: 25vw;
+  width: 50vw;
+  height: 100vh;
+  /* Defines the center of each card (column) as the
+     coordinate that should be used for snapping */
+  scroll-snap-coordinate: 50% 50%;
+}
 
-  margin: 0;
-  transition: transform;
+#cards-view .card.sliding {
+  transition: transform var(--transition-duration);
+}
+
+#cards-view .card > .content {
+  position: absolute;
+  top: 25vh;
+  transition: transform var(--transition-duration);
+  width: 50vw;
+  height: 50vh;
   will-change: transform;
```

Comment:
this is preventing the cardview from being tiled (each card in the viewport gets layer).
I'm not sure we want that and it might explain part of the entering transition lags (with more than 5 appps open)